packages/netsuite-adapter/src/config.ts

```diff
@@ -147,9 +147,9 @@ export const fetchDefault: FetchParams = {
       name: '.*',
     }],
     fileCabinet: [
-      '^/SuiteScripts/.*',
-      '^/Templates/.*',
-      '^/SuiteBundles/.*',
+      '^/SuiteScripts(/.*)?',
```

Comment:
You are correct.
Having said that, maybe the below will be less overwhelming and correct in 99.9% cases (only in case the user has a folder named `/SuiteScripts2/....` it will accidentally fetch it as well). WDYT?
```suggestion
      '^/SuiteScripts.*',
```